ssl/statem/extensions.c

```diff
@@ -1708,6 +2108,12 @@ int tls_psk_do_binder(SSL_CONNECTION *s, const EVP_MD *md,
     OPENSSL_cleanse(finishedkey, sizeof(finishedkey));
     EVP_PKEY_free(mackey);
     EVP_MD_CTX_free(mctx);
+#ifndef OPENSSL_NO_ECH
+    EVP_MD_CTX_free(ctx);
+    if (tpkt_mem != NULL)
+        WPACKET_cleanup(&tpkt);
+    BUF_MEM_free(tpkt_mem);
```

Comment:
As you have an appropriate if already you can put this inside it.

Reply:
same issue as last 3 I think